tests/tests/functional/commands/test_history_deleter.py

```diff
@@ -43,7 +43,7 @@ def test_delete_older_than_weeks():
                             weeks=1, stdout=out)
     # assert
     assert Poll.history.count() == 1
-    assert '1 deleted.' in out.getvalue()
+    assert '1 HistoricalRecord deleted.\n' in out.getvalue()
```

Comment:
Please also add a test for the `ArchivedHistoricalRecord` scenario

Reply:
done